webapp/components/MasonryGrid/MasonryGridItem.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+  <ds-grid-item :rowSpan="rowSpan">
+    <slot></slot>
+  </ds-grid-item>
+</template>
+
+<script>
+export default {
+  data() {
+    return {
+      rowSpan: 10,
+    }
+  },
+  methods: {
+    calculateItemHeight() {
+      this.$parent.$emit('calculating-item-height')
```

Comment:
I was curious what the difference between `this.$emit` and `this.$parent.$emit` was, and found this SO post.
As far as I understand it, this is the way to tell the parent component to emit an event to its parent? so like emit an event to a grandparent component.
Is that correct @alina-beck?

Reply:
Yes, and the reason we have to do it here is that this component sits within a slot – I found the solution in [this comment](https://github.com/vuejs/vue/issues/4332#issuecomment-263444492) and it's working! But hard to test... 😕

Reply:
You could emit again in the parent component
(child)-emit->(parent)-emit->(granddaddy)